ui-lib/src/main/java/co/electriccoin/zcash/ui/common/compose/BalanceWidget.kt

```diff
@@ -40,19 +40,21 @@ private fun BalanceWidgetPreview() {
             modifier = Modifier.fillMaxWidth()
         ) {
             @Suppress("MagicNumber")
-            BalanceWidget(
-                walletSnapshot =
-                    WalletSnapshotFixture.new(
-                        saplingBalance =
-                            WalletBalance(
-                                Zatoshi(1234567891234567),
-                                Zatoshi(123456789),
-                                Zatoshi(123)
-                            )
-                    ),
-                isReferenceToBalances = true,
-                onReferenceClick = {},
-                modifier = Modifier
+            (
+                BalanceWidget(
+                    walletSnapshot =
+                        WalletSnapshotFixture.new(
+                            saplingBalance =
+                                WalletBalance(
+                                    Zatoshi(1234567891234567),
+                                    Zatoshi(123456789),
+                                    Zatoshi(123)
+                                )
+                        ),
+                    isReferenceToBalances = true,
+                    onReferenceClick = {},
+                    modifier = Modifier
+                )
```

Comment:
This is more about my curiosity, I checked the raw file and the only change here is that exactly the same code has been wrapped into parentheses (). Is it by accident or it has some meaning?

Reply:
Having it in parentheses is IMHO not necessary and was caused by Ktlint auto-formtting.